nats-top.yaml

```diff
@@ -0,0 +1,91 @@
+package:
+  name: nats-top
+  version: "0.6.3"
+  epoch: 0
+  description: A top-like tool for monitoring NATS servers.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nats-io/nats-top
+      tag: v${{package.version}}
+      expected-commit: 812d35cc991232d7606816a15b2ea0b12fb129e7
+
+  - uses: go/build
+    with:
+      packages: ./nats-top.go
```

Comment:
We should drop this and have it default to building go.mod, which imports nats-top.go:
 - https://github.com/nats-io/nats-top/blob/main/go.mod

Otherwise we may lose important dependencies which are included in go.mod

Reply:
Noted, I've updated default to building go.mod